tools/cargo-zerocopy/src/main.rs

```diff
@@ -53,6 +53,45 @@ impl fmt::Display for Error {
 
 impl std::error::Error for Error {}
 
+trait CommandExt {
+    fn output_or_exit(&mut self) -> Output;
+    fn execute(&mut self);
+}
+
+impl CommandExt for Command {
+    fn output_or_exit(&mut self) -> Output {
+        if let Ok(output) = self.output() {
+            if !output.status.success() {
+                eprintln!(
+                    "[cargo-zerocopy] failed while capturing output from command: {:?}",
+                    self
+                );
+                let stdout = std::str::from_utf8(&output.stdout).unwrap();
+                let stderr = std::str::from_utf8(&output.stderr).unwrap();
+                eprintln!("[cargo-zerocopy] stdout: {stdout}");
+                eprintln!("[cargo-zerocopy] stderr: {stderr}");
```

Comment:
Would it be possible to prefix `[cargo-zerocopy]` on each line (if stdout/stderr has newlines)?

Reply:
Nvm, I realized that this is only called for tiny utilities like `cargo msrv`, and it's not that important. Not worth the engineering effort IMO, especially given needing to juggle different platforms' end-of-line characters.